tools/project_manager/show_all_good_word_suggestions.js

```diff
@@ -1,7 +1,7 @@
 /*global splitControl */
 
 window.addEventListener("DOMContentLoaded", function () {
-    let mainSplit = splitControl("#suggestions_container", {
+    let mainSplit = splitControl(document.getElementById("suggestions_container"), {
```

Comment:
Just thinking out loud: we may want to add some helper function to shorten all those `document.getElementById` (I find myself adding it whenever I have to do some web work).

Reply:
That would make sense to me. jQuery's selection tool is really nice but we're trying to remove it from the codebase since it's not really needed for the browsers we support anymore. Doesn't mean we can't / shouldn't have some helpers to simplify common things like this.